src/utils/claimgraph.js

```diff
@@ -23,14 +35,23 @@ export function fromJsonldjsCg(jcg) {
 
 // convert a single node from json-ld claimgraph representation to this module's representation.
 function fromJsonldjsNode(jn) {
+  let ret;
   switch (jn.termType) {
     case 'NamedNode':
       return { Iri: jn.value };
     case 'BlankNode':
       return { Blank: jn.value };
     case 'Literal':
       assert(jn.datatype.termType === 'NamedNode', 'The datatype of an RDF literal must be an IRI');
-      return { Literal: { value: jn.value, datatype: jn.datatype.value } };
+      ret = { Literal: { value: jn.value, datatype: jn.datatype.value } };
```

Comment:
you can use `const ret` here, as the reference of `ret` never changes (it remains that object, you are just mutating its values)